### DISCUSSION TITLE
Duda entrega de tp0

### DISCUSSION THREAD

**Discussion by NicoHermosilla** [INGCOM-UNRN-P1/cursada-2023#24] [Preguntas y respuestas] [answered]
Buenas queria consultar como se si mi tp0 fue recibido ya que ayer por la tarde lo envie y nose si tengo que recibir alguna notificacion de que lo entregue o algo

**Comment by DColqueChura** [comment]
Hola! Terminaste y también rellenaste el formulario para entrega de tp -0?

**Reply by NicoHermosilla** [reply]
Hola, si de hecho ayer carge los archivos que pedia al repo y envie el formulario. Mi pregunta es si me tiene que llegar algun correo avisando que le llego mi tp o algo

**Comment by martinvilu** [comment, marked as answer]
Se ve la entrega, te mandé correo por una situación que no afecta la corrección tambien.

Esto está en manos de Miguel.

**Reply by jingusci** [reply]
Tengo la misma consulta, no logro entender cuando el TP está entregado. ¿Como puedo verificarlo?
